hal/include/HAL/Notifier.h

```diff
@@ -14,7 +14,10 @@
 #ifdef __cplusplus
 extern "C" {
 #endif
-HAL_NotifierHandle HAL_InitializeNotifier(void (*process)(uint64_t, HAL_NotifierHandle),
+typedef void (*NotifierProcessFunction)(uint64_t currentTime,
```

Comment:
Want me to add one to the callback in Interrupts too? It doesn't have the prefix either.